mathesar_ui/src/systems/shares/ShareEntity.svelte

```diff
@@ -168,7 +168,12 @@
     {:else if loadRequestStatus.state === 'success'}
       {#if share && share.enabled}
         <InputGroup>
-          <TextInput bind:element={linkInput} readonly value={shareLink} />
+          <TextInput
+            bind:element={linkInput}
+            readonly
+            value={shareLink}
+            style="background: var(--slate-50);"
+          />
```

Comment:
The component will only have a background when CSS is directly applied using inline styles. However, this approach limits its reusability since the style must be passed explicitly each time the component is used elsewhere.